iquip/apps/monitor.py

```diff
@@ -689,13 +944,58 @@ def _setDACVoltage(self, device: str, channel: int, voltage: float):
         """Sets the voltage of the target DAC channel through _DACVoltageThread.
         
         Args:
-            device, channel, voltage: See _DACVoltageThread attributes section.
+            See _DACVoltageThread attributes section.
         """
         self.dacVoltageThread = _DACVoltageThread(
             device, channel, voltage, self.proxy_ip, self.proxy_port
         )
         self.dacVoltageThread.start()
 
-    def frames(self) -> Tuple[TTLControllerFrame, DACControllerFrame]:
+    @pyqtSlot(float, float, float, bool)
```

Comment:
Do you have any reference for this use?
This method itself does not match the signature `(float, float, float, bool)` but `functools.partial()` of it.

Reply:
Umm.. I think it should be changed into `(str, int, float, float, float, bool)`.
Thanks for reviewing it.